packages/core/src/enums/MetadataModules.ts

```diff
@@ -9,12 +9,17 @@
 enum MetadataModules {
   CINE = 'cineModule',
   IMAGE_URL = 'imageUrlModule',
+  SOP_COMMON = 'sopCommonModule',
```

Comment:
It seems that some enums are UpperCamelCase and some are UPPER_UNDERSCORE naming.  Which one do we want to use consistently?   Could you fix this one to UpperCamelCase, since I think that is what we are trying to use?

Reply:
As I see all the enums in the cores/src/enums are actually UPPER_UNDERSCORE, so I don't see any inconsistencies